locales/en/index.yml

```diff
@@ -3220,7 +3220,13 @@ features:
       itemSubtitle: "{{date}} · {{time}} · ID {{id}}"
       signed: "Signed"
       notAvailable: "Not available"
-      signingInProgress: "Signature in progress"   
+      signingInProgress: "Signature in progress"  
+  itWallet:
+    title: "IT Wallet"
+    actionBanner:
+      title: "Novità IT Wallet"
+      description: "Da oggi puoi collezionare le tue credenziali digitali (come Licenza di Guida, Tessera Sanitaria, etc.) e accedere ai servizi online con IT Wallet."
+      action: "Attiva IT Wallet" 
```

Comment:
cc @AliceAzzolini

Reply:
@hevelius in the coming weeks I'll work on the microcopy of the wallet project, even the Italian version is TBD. I'll provide the translation as soon as it's ready

Reply:
Thanks @AliceAzzolini! In the meantime, in order not to affect the releases, these pr's will be merged anyway. We will return to the copy and translations as soon as possible.

Reply:
Sure :)